.gitignore

```diff
@@ -106,3 +106,6 @@ venv.bak/
 # IDEs
 .vscode/
 .idea/
+
+# profiles files generated by pytest-profiling
+/prof/
```

Comment:
Did the output help you? Looks quite nice on their website.

Reply:
As soon as gevent is involved, the output is messed up. For now, `pytest --durations` is more helpful.